src/scicat_dataset.py

```diff
@@ -481,24 +489,13 @@ def _create_scientific_metadata(
 
     """
     return {
-        # Default field
-        "ingestor_metadata_schema_id": {
-            "value": metadata_schema_id,
-            "unit": "",
-            "human_name": "Ingestor metadata schema ID",
-            "type": "string",
```

Comment:
We don't need this default values anymore?

Reply:
We discussed it offline and Max decided to remove it in case other facilities want to use the ingestor as well.

Reply:
And the default ones will be in included in the metadata parsing configurations.